Add clarification where X-Guillotine-SiteKey can be used #869 (#870)
enonic/app-guillotine: docs/usage.adoc

=== Path placeholder

Use the `${site}` placeholder when querying for paths within a site. This placeholder will be replaced with an actual path of the site upon query resolution.

> NOTE: `${site}` placeholder is supported for the fields `get`, `getChildren`, `getChildrenConnection`, `getPermissions` and `getSite` of the <<api#headlessCms, HeadlessCms>> type. The `getPermissions` and `getSite` fields do not have a `key` argument, and current content is determined by the `X-Guillotine-SiteKey` header or the `siteKey` argument of the `guillotine` field. The same applies to the `get`, `getChildren`, and `getChildrenConnection` fields if the `key` argument is not provided.

Reviewer: ```suggestion
NOTE: `${site}` placeholder is supported for the fields `get`, `getChildren`, `getChildrenConnection`, `getPermissions` and `getSite` of the <<api#headlessCms, HeadlessCms>> type. The `getPermissions` and `getSite` fields do not have the `key` argument, and current content is determined by the `X-Guillotine-SiteKey` header or the `siteKey` argument of the `guillotine` field. The same applies to the `get`, `getChildren`, and `getChildrenConnection` fields if the `key` argument is not provided.
```

> Use the `${site}` placeholder when querying for path's within the site.

Reviewer: ```suggestion
Use the `${site}` placeholder when querying for paths within a site. This placeholder will be replaced with an actual path of the site upon query resolution.
```

> NOTE: This substitutes the placeholder with the actual path to the site and is supported for the fields `get`, `getChildren`, `getChildrenConnection`, `getPermissions` and `getSite` of the <<api#headlessCms, HeadlessCms>> type.

Reviewer: ```suggestion
NOTE: `${site}` placeholder is supported for the fields `get`, `getChildren`, `getChildrenConnection`, `getPermissions` and `getSite` of the <<api#headlessCms, HeadlessCms>> type.
```

> NOTE: `${site}` placeholder is supported for the fields `get`, `getChildren`, `getChildrenConnection`, `getPermissions` and `getSite` of the <<api#headlessCms, HeadlessCms>> type. The `getPermissions` and `getSite` fields do not have a `key` argument, and the current content is determined by the `X-Guillotine-SiteKey` header or the `siteKey` argument of the guillotine. The same applies to the `get`, `getChildren`, and `getChildrenConnection` fields if the `key` argument is not provided.

Reviewer: ```suggestion
NOTE: `${site}` placeholder is supported for the fields `get`, `getChildren`, `getChildrenConnection`, `getPermissions` and `getSite` of the <<api#headlessCms, HeadlessCms>> type. The `getPermissions` and `getSite` fields do not have a `key` argument, and current content is determined by the `X-Guillotine-SiteKey` header or the `siteKey` argument of the `guillotine` field. The same applies to the `get`, `getChildren`, and `getChildrenConnection` fields if the `key` argument is not provided.
```
